xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -784,6 +789,11 @@ void CMusicInfoTag::SetSongVideoURL(const std::string& songVideoURL)
   m_songVideoURL = songVideoURL;
 }
 
+void CMusicInfoTag::SetChapterMarks(const std::map<int, chapterDetails>& chapters)
+{
+  m_chapters = chapters;
+}
+
```

Comment:
Because impl. is trivial, I would make this method definition inline in the header.

Reply:
As above

Reply:
Up to you of course.